src/app.js

```diff
@@ -0,0 +1,20 @@
+const http = require('http')
+const { formGenerator } = require('./tool.js')
+
+const formInputs = [
+  {
+    type: 'text' 
+  }, {
+    type: 'email'
+  }, {
+    type: 'number'
+  }, {
+    type: 'submit'
+  }
+]
+
+http.createServer(function(request, response) {  
+  response.writeHeader(200, {"Content-Type": "text/html"});  
+  response.write(formGenerator(inputs));
```

Comment:
A better implementation would be to accept a POST request accepting an array that processes and outputs the html of your tool. This implementation would be minimal pivot from your current implementation.